build.sbt

```diff
@@ -10,32 +10,28 @@ val Scala212 = "2.12.13"
 val Scala213 = "2.13.4"
 
 val AkkaBinaryVersionForDocs = "2.6"
-val KafkaVersionForDocs = "26"
+val KafkaVersionForDocs = "27"
 
-val akkaVersion = "2.6.10"
-val kafkaVersion = "2.6.0"
-// TODO Jackson is now a provided dependency of kafka-clients
-// https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients/2.6.0
-val jacksonVersion = "2.10.5.1"
+val akkaVersion = "2.6.14"
+val kafkaVersion = "2.7.0"
+// Jackson is now a provided dependency of kafka-clients
+// This should align with the Jackson minor version used in Akka 2.6.x
+// https://github.com/akka/akka/blob/master/project/Dependencies.scala#L23
+val jacksonVersion = "2.11.4"
```

Comment:
Add a reminder for the release notes as kafka-clients 2.7.0 is still on Jackson 2.10.5.

Reply:
I noticed that as well. Since the tests seem to work with this pulled up version I think it's fine to use this. What would include in the release notes? "Akka uses 2.11, Kafka refs (provided) 2.10. Akka uses 2.11 because ____"